Library/Homebrew/cmd/cleanup.rb

```diff
@@ -64,12 +64,13 @@ def cleanup_formula f
   end
 
   def cleanup_keg keg
+    size = `/usr/bin/du -sh #{keg} | /usr/bin/awk -F'[^0-9]*' '{print $2}'`.strip
     if keg.linked?
       opoo "Skipping (old) #{keg} due to it being linked"
     elsif ARGV.dry_run?
-      puts "Would remove: #{keg}"
+      puts "Would remove: #{keg}, #{size}M"
     else
-      puts "Removing: #{keg}..."
+      puts "Removing: #{keg}, #{size}M"
```

Comment:
There's a discussion about the style in the general thread. Wasn't sure whether we wanted the style from ` install ` or from ` info `. Looks like parenthesis is the popular winner so far.